index.html

```diff
@@ -14,22 +14,35 @@
 <body>
   <section class="container">
   <!-- header section  -->
+  <header>
+  <div id="nav" class="navhidden">
+    <div class="topmost-bar">
+      <ul>
+         <li><i class="fa fa-facebook"></i></li>
+         <li><i class="fab fa-twitter"></i></li>
+         <li><a href="#">English</a></li>
+         <li><a href="#">My Page</a></li>
+         <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    </div>
   <nav class="navbar">
     <div class="navhidden">
       <span class="navspan"><i  class="fa fa-cutlery"  aria-hidden="true"></i>HB-DIET
```

Comment:
- [ ] [Optional] You con add the link to the home page to the logo, this will make your home page more accessible from the about page